tests/phpunit/Unit/functions/StrToKebabCaseTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Cig\Tests\Unit\Functions;
+
+
+class StrToKebabCaseTest extends \Cig\Tests\Unit\BaseTestCase {
+
+	/**
+	 * @covers ::\Cig\str_to_kebab_case()
+	 */
+	public function test_str_to_kebab_case(): void {
+		$string = "welcome~ to *kebab case!!";
+		$expected_result = "welcome-to-kebab-case";
+
+		//this method uses another method (str_to_words) from same file
+		$result = \Cig\str_to_kebab_case($string);
```

Comment:
Another place we should stub `str_to_words`. Really most cases the function being tested is using another method/function wee should stub the other calls.

Due to the way, we've been doing development we'll for sure run into cases where once the other calls are stubbed the testing will feel kind of silly, but that should likely help show us how to write code in a more testable way.